master/internal/sproto/resources.go

```diff
@@ -153,14 +201,14 @@ func NewResourcesFailure(
 	}
 }
 
-func (f ResourcesFailure) Error() string {
-	if f.ExitCode == nil {
-		if len(f.ErrMsg) > 0 {
-			return fmt.Sprintf("%s: %s", f.FailureType, f.ErrMsg)
+func (r ResourcesFailure) Error() string {
```

Comment:
No reason to change existing ones; just make the new ones match.

Reply:
I changed this because for some reasons, I couldn't pass the linter tests. But let me change it back to see what happen.